src/sp/it/pl/util/dev/Fail.kt

```diff
@@ -0,0 +1,38 @@
+@file:JvmName("Fail")
```

Comment:
About JvmName for files/classes. Id like it if no kotlin class was named without the Kt suffix. Otherwise Java consumer will be confused as to when they call Kotlin code.

Reply:
So you propose to not use the JvmName annotation?

Reply:
Yeah, unless we have a good reason to have a different class name for Java, but since that is confusing, I can not imagine a use case right now. I think Kotlin guys made a good decision to have Java see the Kt suffix.